src/DynamoCore/Graph/NodeGraph.cs

```diff
@@ -13,13 +13,40 @@
 
 namespace Dynamo.Graph
 {
+    /// <summary>
+    /// This class is used to load workspace from xml file.
+    /// It loads in Dynamo next items: Nodes, Connectors, Notes etc.
+    /// </summary>
     public class NodeGraph
     {
+        /// <summary>
+        /// Nodes loaded from xml.
+        /// </summary>
```

Comment:
Put a cref to Nodes, connectors and Annotations. that cref should take the user to Nodemodel / connectormodel / annotationmodel.
it should say NodeModels loaded from XML. Clicking NodeModels should take them to NodeModel class. Do this for the other ones.